README.md

```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
 ***
 
 ## Deployment
+1. Download docker image `phenoscape/pipeline-tools` from [here](https://hub.docker.com/r/phenoscape/pipeline-tools).
+2. Execute `run.sh` to launch the container. Pass in commands to build different targets as arguments to `run.sh`. Pass `make all` to run the entire pipeline. 
```

Comment:
The way this is phrased makes it sounds like you execute `run.sh` and then pass in commands. Just give a complete working example command line.